docs/conf.py

```diff
@@ -222,6 +226,7 @@
           p.appendChild(document.createTextNode('.'));
         }
       </script>
+      -->
```

Comment:
Let's just get rid of this whole `<script>` block rather than commenting it out!